pkgs/applications/editors/emacs/elisp-packages/update-from-overlay.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+
+import contextlib
+import datetime
+import os
+import shutil
+import subprocess
+import sys
+import urllib
+
+# External Dependencies:
+import github
+import pygit2
```

Comment:
another choice is [GitPython](https://github.com/gitpython-developers/GitPython), maybe more pythonic